app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
Don't touch **SDK version**. Termux is not compatible with anything beyond 28.

Reply:
I'll see if it can be done in version 28, but at a glance it seems like all the sim slot stuff is API version 29. I bumped the version up because [``getSubscriptionIds``](https://developer.android.com/reference/android/telephony/SubscriptionManager#getSubscriptionIds(int)), which is used to setup the ``SmsManager`` for a given slot, was added in API level 29.